smif/sos_model.py

```diff
@@ -553,6 +538,9 @@ def __init__(self, models, sos_model):
         self._model_names = {model.name for model in models}
         self._sos_model = sos_model
         self.iterated_results = {}
+        # tolerance for convergance assessment - see numpy.allclose docs
```

Comment:
Should these be exposed to config as in `max iterations`? Environment variables?